activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -902,6 +902,28 @@ def down
       $stdout = original
     end
   end
+
+  def test_schema_dump_keeps_enum_intact_if_it_contains_comma
+    original, $stdout = $stdout, StringIO.new
+
+    migration = Class.new(ActiveRecord::Migration::Current) do
+      def up
+        create_enum "enum_with_comma", ["value1", "value,2", "value3"]
+      end
+
+      def down
+        drop_enum "enum_with_comma"
+      end
+    end
+
+    migration.migrate(:up)
+    output = dump_all_table_schema
+
+    assert_match %r{create_enum "enum_with_comma", \["value1", "value,2", "value3"\]}, output
+  ensure
+    migration.migrate(:down) if migration
+    $stdout = original
+  end if current_adapter?(:PostgreSQLAdapter)
```

Comment:
```suggestion
  if current_adapter?(:PostgreSQLAdapter)
    def test_schema_dump_keeps_enum_intact_if_it_contains_comma
      original, $stdout = $stdout, StringIO.new

      migration = Class.new(ActiveRecord::Migration::Current) do
        def up
          create_enum "enum_with_comma", ["value1", "value,2", "value3"]
        end

        def down
          drop_enum "enum_with_comma"
        end
      end

      migration.migrate(:up)
      output = dump_all_table_schema

      assert_includes output, 'create_enum "enum_with_comma", ["value1", "value,2", "value3"]'
    ensure
      migration.migrate(:down) if migration
      $stdout = original
    end
  end
```

Reply:
Right. Unfortunately, this raises `ActiveSupport::RaiseWarnings::WarningError: Test is missing assertions: (...)` when testing a different adapter.

Reply:
Even with `if current_adapter?` at the beginning, as suggested?

Reply:
I see. Either way works, but I chose to follow the style I found in the file, [as suggested](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#follow-the-coding-conventions).

> Follow the conventions in the source you see used already.

Reply:
Line 313, for example. Before marking the conversation as resolved, take a look at other changes I suggested in the diff.

Reply:
I appreciate your willingness to help @fatkodima, I really do, but I think I'm missing your point. In this codebase, is nesting methods in an if statement preferred over writing it inline?

Reply:
Nevermind, I just saw another block for the PG adapter and put it in there. It does look cleaner. Let me know what you think. Thanks.